doc/src/installation.rst

```diff
@@ -186,3 +186,19 @@ directory which is on the library search path.  Alternatively, the
 paths can be specified explicitly by exporting the environment
 variables ``PYFR_METIS_LIBRARY_PATH=/path/to/libmetis.so`` and/or
 ``PYFR_SCOTCH_LIBRARY_PATH=/path/to/libscotch.so``.
+
+Conduit/Ascent
+^^^^^^^^^^^^^^
+
+Detailed instructions on the compilation and install of
+`Conduit <https://llnl-conduit.readthedocs.io>`_ and
+`Ascent <https://ascent.readthedocs.io>`_ can be found in the respective
+documentation. MPI and VTK-m support is currently required for the plugin. In
+order for PyFR to find the libraries it may be necessary to export the library
+paths. For example, on linux you will need::
+
+    export PYFR_CONDUIT_LIBRARY_PATH=/path/to/libconduit.so
+    export PYFR_ASCENT_MPI_LIBRARY_PATH=/path/to/libascent_mpi.so
+
+Currently the plugin requires that Ascent and Conduit are 64-bit, this is
+default when compiling in most cases.
```

Comment:
Is there a minimum version number requirement for Ascent, and thus for Conduit and VTK-m?